### PR TITLE
Possible fix for incorrect host being used

### LINKED ISSUE #103: Console output hanging on polling when tested against https URL
Decided to test against BMA environments to make sure auditjs-iq functions against a separate https iq server endpoint

It looks like it is hanging on the polling and not getting confirmation back from the IQ Server that the scan is done

`ajurgenson@ArtieSonaDell:~/test-apps/Angular-JumpStart$ auditjs iq -a auditjs-uat -s build

✔ Starting application
✔ Getting coordinates for Nexus IQ Server
✔ Auditing your application with Nexus IQ Server
✔ Getting your internal application ID
✔ Submitting your dependencies to Nexus IQ Server
⠴ Checking for results (this could take a minute)`

That's where it's hanging indefinitely, but the report is available in IQ Server pretty quickly.  

Will test against internal IPs through the vpn to see if this is an https issue or something else

### PR DESCRIPTION
**Pull request by allenhsieh**
requestService.asyncPollForResults was using the incorrect argument for host, i've updated it to use the requestService's host. 

Closes #103

### CODE REVIEW

File: src/Application/Application.ts [hunk 1]
```diff
@@ -212,7 +212,7 @@ export class Application {
       this.spinner.maybeSucceed();
       this.spinner.maybeCreateMessageForSpinner('Checking for results (this could take a minute)');
       logMessage('Polling Nexus IQ Server for report results', DEBUG, resultUrl);
-      requestService.asyncPollForResults(`${args.server}/${resultUrl}`, (x) => {
+      requestService.asyncPollForResults(`${requestService.host}/${resultUrl}`, (x) => {
```
Comment (DarthHater) [inline src/Application/Application.ts]: This is a great catch, but we probably should just pass in the `resultURL` and then do the append of the host inside the RequestService, to likely avoid some stupid bug that Jeffry created like this in the near future :)

Comment (ajurgenson55) [reply src/Application/Application.ts]: tested it and it works now, will merge since its breaking and we can look at the stylistic stuff in the beta

Comment (allenhsieh) [reply src/Application/Application.ts]: @ajurgenson55 Sounds good! I was investigating how to best implement @DarthHater's request, i'll open it as an issue to fix on another branch.

**Pull request merged**